packages/cli/src/util/deploy/create-git-meta.ts

```diff
@@ -0,0 +1,155 @@
+import fs from 'fs-extra';
+import { join } from 'path';
+import ini from 'ini';
+import git from 'git-last-commit';
+import {
+  BitbucketMeta,
+  GitHubMeta,
+  GitLabMeta,
+  GitMeta,
+  RepoData,
+} from '../../types';
+import { Output } from '../output';
+
+function getLastCommit(directory: string): Promise<git.Commit> {
+  return new Promise((resolve, reject) => {
+    git.getLastCommit(
+      (err, commit) => {
+        if (err) return reject(err);
+        resolve(commit);
+      },
+      { dst: directory }
+    );
+  });
+}
+
+export async function getRepoData(configPath: string, output: Output) {
```

Comment:
It might be good to add return types of these functions so you catch any missing/incorrect returns early.

Reply:
Thanks for catching this!